weepickle/test/src/com/rallyhealth/weepickle/v1/ClassDefs.scala

```diff
@@ -192,6 +192,10 @@ object Annotated {
   object C {
     implicit def rw: RW[C] = WeePickle.macroFromTo
   }
+  case class D(a: String, @jsonIgnore b: Option[String], c: String)
```

Comment:
How the case class instance can be parsed parsed back if `@jsonIgnore` or `@dropAlways` is used for a non-optional field without default values?

Reply:
Yes, I was wondering about this, whether it should be ignored on both serialization and deserialization, or require a default or `Option` type. I've previously used Jackson's `@JsonIgnore` annotation for fields that a team's frontend consumers no longer supported, but another team's direct backend consumers still used (in a monorepo). I'll have to check my current use case: [lib-pokitdok-client#PokitdokResponse](https://github.com/AudaxHealthInc/lib-pokitdok-client/blob/3fc36451ab1945727c6c9763e421c41c2bd14ac0/play23/src/main/scala/com/rallyhealth/pokitdok/v10/PokitdokRequestBuilder.scala#L139-L158) has a lot of redacted fields with Play Json...

Reply:
@dropAlways, not @jsonIgnore? Note that @dropAlways is more generic -- you may be serializing to YAML and not Json.

Reply:
Yeah, Doug recommended I be more generic since it looks like these macros handle several serialization types. Would make behavior more consistent as well across types.

Reply:
The macro doesn't have any idea if your case class will become json, yaml, msgpack, cbor, xml, csv, etc.

We already have `@dropDefault`. Reusing the "drop" term made sense to me here, since it's a similar effect.